justfile

```diff
@@ -13,14 +13,19 @@ dotenv:
     @([ ! -f openverse_catalog/.env ] && cp openverse_catalog/env.template openverse_catalog/.env) || true
 
 
-up: dotenv
-    docker-compose {{ DEV_DOCKER_FILES }} up -d
+up flags="": dotenv
```

Comment:
Does adding `flags=""` allow us adding parameters? It would be awesome to have that for tests, too, then. This way, we could debug failing tests by only running that single test.

Reply:
Oh yeah! That's simple enough to add, I'll push that in a second.